File: LATIHAN B/PHP/TimSepakBola.php
<?php

class TimSepakBola
{
    //atribut privat
    private $NamaTim = "Lorem Ipsum";
    private $AsalNegaraTim = "Indonesia";
    private $NamaPemain = "Ipsum Lorem";
    private $TahunBerdiri = 1;
    
    //construct
    public function __construct($NamaTim = "Lorem Ipsum", $AsalNegaraTim = "Indonesia", $NamaPemain = "Ipsum Lorem", $TahunBerdiri = 2000){
        $this->NamaTim = $NamaTim;
        $this->AsalNegaraTim = $AsalNegaraTim;
        $this->NamaPemain = $NamaPemain;
        $this->TahunBerdiri = $TahunBerdiri;
    }
    
    /*get set*/
    public function setNamaTim($NamaTim){
        $this->NamaTim = $NamaTim;
    }

    public function getNamaTim(){
        return $this->NamaTim;
    }

    public function setAsalNegara($AsalNegaraTim){
        $this->AsalNegaraTim = $AsalNegaraTim;
    }

    public function getAsalNegara(){
        return $this->AsalNegaraTim;
    }

    public function setNamaPemain($NamaPemain){
        $this->NamaPemain = $NamaPemain;
    }

    public function getNamaPemain(){
        return $this->NamaPemain;
    }

    public function setTahunBerdiri($TahunBerdiri){
        $this->TahunBerdiri = $TahunBerdiri;
    }

    public function getTahunBerdiri(){
        return $this->TahunBerdiri;
    }

}

//fungsi untuk menampilkan data
function tampilData($TimSepakBola){
    echo "Data Tim Sepak Bola" ."<br>";
    echo "Nama Tim: " .$TimSepakBola->getNamaTim() ."<br>";
    echo "Asal Negara Tim: " .$TimSepakBola->getAsalNegara() ."<br>";
    echo "Nama Pemain: " .$TimSepakBola->getNamaPemain() ."<br>";
    echo "Tahun Tim Didirikan: " .$TimSepakBola->getTahunBerdiri() ."<br> ======== <br>";
}

?>

File: LATIHAN B/CPP/TimSepakBola.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string>

using namespace std;

//bismillahirrahmanirrahiim
/*
Nama : Sabian Annaya Havid
Program Studi/Kelas : Ilmu Komputer/C2
NIM : 2005021

Saya Sabian Annaya Havid mengerjakan Latihan dalam mata kuliah
Desain Pemograman Berorientasi Objek untuk keberkahanNya maka saya tidak melakukan kecurangan
seperti yang telah dispesifikasikan. Aamiin.
*/

class TimSepakBola{
    private:
        /*attributes*/
        string namaTim;
        string asalNegara;
        int tahunBerdiri;
        string namaPemain;
    
    public:
        //constructor
        TimSepakBola(){
            this->namaTim = this->namaPemain = "test";
            this->tahunBerdiri = 9999;    
        }

        //constructor, but with parameter
        TimSepakBola(string namaTim, string asalNegara, int tahunBerdiri, string namaPemain){
            this->namaTim = namaTim;
            this->asalNegara = asalNegara;
            this->tahunBerdiri = tahunBerdiri;
            this->namaPemain = namaPemain;
        }  

        /* get - set for each attributes */
        void setNamaTim(string namaTim){
            this->namaTim = namaTim;
        }

        string getNamaTim(){
            return namaTim;
        }

        void setAsalNegaraTim(string asalNegara){
            this->asalNegara = asalNegara;
        }

        string getAsalNegaraTim(){
            return asalNegara;
        }

        void setNamaPemain(string namaPemain){
            this->namaPemain = namaPemain;
        }

        string getNamaPemain(){
            return namaPemain;
        }

        void setTahunBerdiri(int tahunBerdiri){
            this->tahunBerdiri = tahunBerdiri;
        }

        int getTahunBerdiri(){
            return tahunBerdiri;
        }

        /*Destructor*/
        ~TimSepakBola(){
        }

};
